ose/utils.py

```diff
@@ -0,0 +1,21 @@
+"""A set of utilities to work with xAPI and other data formats."""
+
+import json
+import os
+
+
+def load_agent_data(filename):
+
+    data = []
+    with open(filename) as f:
+        for line in f.readlines():
+            data.append(json.loads(line))
+    return data
+
+
+if __name__ == "__main__":
+    base_path = os.path.abspath(
```

Comment:
Maybe move base_path and into load_agent_data

Please keep the coverage up :o)